app/models/energy/ScheduleBase.py

```diff
@@ -0,0 +1,60 @@
+"""Schedule Base Schema"""
+from pydantic import BaseModel, Schema, validator, ValidationError, UrlStr, ConstrainedStr
+from typing import List, Union
+from enum import Enum
+from uuid import UUID, uuid4
+from datetime import datetime
+
+
+class UnitType (str, Enum):
```

Comment:
Seems like this should be in a module or something to make it specific to schedules.  Either Schedule::UnitType or ScheduleUnitType